src/core/public/http/fetch.test.ts

```diff
@@ -481,6 +481,21 @@ describe('Fetch', () => {
 
       expect(ndjson).toEqual(content);
     });
+
+    it('should make requests for text/event-stream content', async () => {
+      const body = 'data: ';
+      fetchMock.post('*', () => ({
+        body,
+        headers: { 'Content-Type': 'text/event-stream' },
+      }));
+
+      const data = await fetchInstance.post('/my/path', {
+        body,
+        asResponse: true,
+      });
+
+      expect(await data.response?.text()).toEqual(body);
```

Comment:
Nit: Do we need to check if `data.response.body` was returned as expected here?

Reply:
Sure, added one more assertion data.response.body === data.body.